recipes/pngcheck/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "4.0.0" %}
+
+package:
+  name: pngcheck
+  version: {{ version }}
+
+source:
+  url: https://github.com/pnggroup/pngcheck/archive/v{{ version }}.tar.gz
+  sha256: ed13f49bc1205bdf7cd0fc208b6e0eef550da021d1631f7180f718a4db379398
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - cmake
+    - make
```

Comment:
The Windows recipe uses `ninja`, but it's not installed here. Why not use `ninja` on both, unix and win?

Reply:
hmm, yeah.... because mostly i let my muscle memory complete things.

lets try